shop/Customer.java

```diff
@@ -0,0 +1,14 @@
+package shop;
+
+public class Customer extends Person {
+
+    private int CID;
```

Comment:
only final variables should be all-uppercase

Reply:
give more understandable variable name
```suggestion
    private int customerID;
```